src/bp-core/bp-core-attachments.php

```diff
@@ -1450,8 +1575,33 @@ function bp_attachments_cover_image_ajax_upload() {
 		);
 	}
 
-	$cover_subdir = $object_data['dir'] . '/' . $bp_params['item_id'] . '/cover-image';
-	$cover_dir    = trailingslashit( $bp_attachments_uploads_dir['basedir'] ) . $cover_subdir;
+	/**
+	 * Filters BuddyPress image attachment sub directory.
+	 *
+	 * @since BuddyBoss [BBVERSION]
+	 *
+	 * @param string $subdir     The sub dir to uploaded BuddyPress image.
+	 * @param string $object_dir The object dir (eg: members/groups). Defaults to members.
+	 * @param int    $item_id    The object id (eg: a user or a group id). Defaults to current user.
+	 * @param string $type       The type of the attachment which is also the subdir where files are saved.
+	 *                           Defaults to 'cover-image'
+	 */
+	$cover_subdir = apply_filters( 'bp_attachments_get_attachment_sub_dir', $object_data['dir'] . '/' . $bp_params['item_id'] . '/cover-image', $object_data['dir'], $bp_params['item_id'], 'cover-image' );
```

Comment:
```suggestion
	$cover_subdir = apply_filters( 'bb_attachments_get_attachment_sub_dir', $object_data['dir'] . '/' . $bp_params['item_id'] . '/cover-image', $object_data['dir'], $bp_params['item_id'], 'cover-image' );
```

Newly introduced filters should be start with `bb_` instead of `bp_`